HousingSearchApi.Tests/V1/E2ETests/Steps/GetAssetSteps.cs

```diff
@@ -170,12 +187,28 @@ public async Task ThenNumberOfBedSpacesShouldBeInResult(int numberOfBedSpaces)
 
             result.Results.Assets.All(x => x.AssetCharacteristics.NumberOfBedSpaces == numberOfBedSpaces);
         }
+
         public async Task ThenFloorNoShouldBeInResult(string floorNo)
         {
             var resultBody = await _lastResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
             var result = JsonSerializer.Deserialize<APIResponse<GetAssetListResponse>>(resultBody, _jsonOptions);
```

Comment:
Response type from `/all` end point is `APIAllResponse<GetAllAssetListResponse>`, so would be best to reflect that in the test to ensure any future updates to the types won't cause any issues. This applies to all tests against `/all` end point.